pymodbus/server/async_io.py

```diff
@@ -556,19 +551,14 @@ class _serverList:
     :meta private:
     """
 
-    active_server: ModbusTcpServer | ModbusUdpServer | ModbusSerialServer
-
-    def __init__(self, server):
-        """Register new server."""
-        self.server = server
-        self.loop = asyncio.get_event_loop()
```

Comment:
We do need to register the server, that is sort of the whole purpose of the class (keep track of multiple servers).

it used in line 596 and can not be undefined (its interesting that the linters do not complain in line 596, since the class no longer have a loop variable.

Reply:
They do not complain because each server has already has `self.loop`

https://github.com/pymodbus-dev/pymodbus/blob/fe809d9685a15f2b476b396d1dbee0cfca93b56f/pymodbus/server/async_io.py#L267

Reply:
aah...but we still need to register the server (or somehow have a list somewhere).